### ISSUE TITLE
Reset device configuration

### ISSUE THREAD

**Issue by thiagoa** [libratbag/piper#535]
It would be great if there was a button to reset a device to default settings. I'm completely lost trying to find documentation about how to do that.

Sometimes I change things too much but do not remember what the default actions were. With my MX Master 3 mouse (aside from having problems to map a few buttons, which is related to a libratbag issue), one of the buttons was mapped to the "Unknown" label, which triggered Super + tab in a very particular way that I can't reproduce with the available options.

Now I simply can't reset the mouse to default settings and I'm having a hard time trying to find out how to do that. I could not yet find where libratbag or piper store their settings to remove them and start over.

**Comment by stephanlachnit** [comment]
I think this isn't even implemented in libratbag, @FFY00?

**Comment by thiagoa** [comment]
@stephanlachnit No, I mean resetting to the "factory" (or OS) settings for a device. For example, I changed my configuration many times and now I want to see how it looks like by default.

Do you know where ratbagctl or piper store their configuration so that I can delete it and thus revert it?

**Comment by FFY00** [comment]
This is not implemented and probably won't.

We can implement saving/loading profiles in piper, that should fix the issue.

**Comment by thiagoa** [comment]
In the meantime, here's what I did to fix my problem if anyone's looking for a solution, as the examples given by `ratbagctl` are kind of misleading:

```bash
$ ratbagctl list # Get NAME_OF_THE_DEVICE
$ ratbagctl NAME_OF_THE_DEVICE profile active get # Get ACTIVE_PROFILE_ID
$ ratbagctl NAME_OF_THE_DEVICE profile ACTIVE_PROFILE_ID disable
```

 Open up Piper and the notice the configurations are set back to default.

**Comment by FFY00** [comment]
That makes sense, but is a Logitech specific behavior.

**Comment by RoseIynn** [comment]
> In the meantime, here's what I did to fix my problem if anyone's looking for a solution, as the examples given by `ratbagctl` are kind of misleading:
> 
> ```shell
> $ ratbagctl list # Get NAME_OF_THE_DEVICE.
> $ ratbagctl NAME_OF_THE_DEVICE profile active get # Get ACTIVE_PROFILE_ID.
> $ ratbagctl NAME_OF_THE_DEVICE profile ACTIVE_PROFILE_ID disable
> ```
> 
> The command for me was:
> 
> ```shell
> $ ratbagctl hollering-marmot profile 0 disable
> ```
> 
> Open up Piper and notice the configurations are set back to default.

I'm sorry, but I'm confused. How do I get my active profile ID?

**Comment by thiagoa** [comment]
> > In the meantime, here's what I did to fix my problem if anyone's looking for a solution, as the examples given by `ratbagctl` are kind of misleading:
> > ```shell
> > $ ratbagctl list # Get NAME_OF_THE_DEVICE.
> > $ ratbagctl NAME_OF_THE_DEVICE profile active get # Get ACTIVE_PROFILE_ID.
> > $ ratbagctl NAME_OF_THE_DEVICE profile ACTIVE_PROFILE_ID disable
> > ```
> > 
> > 
> > The command for me was:
> > ```shell
> > $ ratbagctl hollering-marmot profile 0 disable
> > ```
> > 
> > 
> > Open up Piper and notice the configurations are set back to default.
> 
> I'm sorry, but I'm confused. How do I get my active profile ID?

Sure. First, run `ratbagctl list` to get the name of your device. After that, run `ratbagctl NAME_OF_THE_DEVICE profile active get`. Replace `NAME_OF_THE_DEVICE` with the device from the first command.
